game/src/Game/GameStart.gd

```diff
@@ -67,23 +72,40 @@ func _load_compatibility_mode() -> void:
 			# and if not also search for a Steam install
 			actual_base_path = GameSingleton.search_for_game_path("..")
 		if not actual_base_path:
-			OS.alert(tr("ERROR_ASSET_PATH_NOT_FOUND_MESSAGE"), tr("ERROR_ASSET_PATH_NOT_FOUND"))
-			get_tree().quit()
-			return
+			get_tree().paused = true
+			vic2_dir_dialog.popup_centered_ratio()
+			# Remove with https://github.com/godotengine/godot/pull/81178
+			vic2_dir_dialog.ok_button_text = "VIC2_DIR_DIALOG_SELECT"
+			# WHY WON'T CANCEL AUTO-TRANSLATE WORK NOW?!?!?!?
+			var cancel_button := vic2_dir_dialog.get_cancel_button()
+			cancel_button.auto_translate = false
+			cancel_button.auto_translate = true
+			var failure_func := func() -> void:
+				get_window().mode = Window.MODE_WINDOWED
+				OS.alert(tr("ERROR_ASSET_PATH_NOT_FOUND_MESSAGE"), tr("ERROR_ASSET_PATH_NOT_FOUND"))
+				get_tree().quit()
+			vic2_dir_dialog.canceled.connect(failure_func)
+			await vic2_dir_dialog.dir_selected
+			get_tree().paused = false
+			actual_base_path = GameSingleton.search_for_game_path(vic2_dir_dialog.current_path)
```

Comment:
If the selected path doesn't work, this will default back to finding the Vic2 Steam install. For now this is fine, since if a Steam install could be found the earlier `search_for_game_path` would have got it, but in the future if we use this for a manual "switch Vic2 install directory" option we might want the ability to limit the search to only the hint path so the user sees an error message if they select an invalid folder.